prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,17 @@
+const languages = [
+  'html',
+  'css',
+  'javascript',
+  'react',
+  'node',
+  'postgres'
+];
+console.log(languages)
+const firstElement = languages [0]
+console.log('the first element of the array is an', firstElement)
+const thirdElement = languages [2]
+console.log('the 3rd element of the array is', thirdElement)
+const length = languages.length;
+console.log('the length of the array is', length)
+const lastElement = languages [6 - 1]
```

Comment:
instead of using the number `6` here, you want to make use of the `length` variable you created on line 10 for this calculation. Let me know if you have any questions!